Registry/Registry.EM_COMMON

```diff
@@ -3111,6 +3120,17 @@ package	ltng_crm_PR92w      lightning_option==1            -       state:ic_flas
 package ltng_crm_PR92z      lightning_option==2            -       state:ic_flashcount,ic_flashrate,cg_flashcount,cg_flashrate
 package	ltng_cpm_PR92z      lightning_option==11           -       state:ic_flashcount,ic_flashrate,cg_flashcount,cg_flashrate
 package	ltng_lpi            lightning_option==3           -        state:lpi
+rconfig  integer    dyn_lightning_option         namelist,physics   max_domains    0      h    "dyn_lightning_option"          "Dynamic Lightning CG-IC Stroke Scheme"       ""
```

Comment:
@BarryHLynn 
Barry,
If you are using the Registry.EM_COMMON, it is more conventional to have the packages listed in the package area, to have the rconfigs listed in the rconfig area. Alternatively, you can have your own registry file, and that is included in registry.em_shared_collection.